### ISSUE TITLE
Please add an option to not require image

### ISSUE THREAD

**Issue by psytechno604** [DangerOnTheRanger/maniwani#71]
Or, at least - save user's message if he fails to attach an image. It is very frustrating to get data loss...

**Comment by DangerOnTheRanger** [comment]
Yes, I've been meaning to work on saving everything in the form in the case of incorrect CAPTCHA/etc. - definitely something that needs to get added.
